File: typescript/basic_grammar/operation/assignment_operator/multiplication.ts
export const multiplicationAssignmentOperator = () => {
  let a = 1
  for (let i = 0; i <  5; i++) {
    a *= 2
  }
  return a
}

export const decimalMultiplicationAssignmentOperator = () => {
  let a = 1
  for (let i = 0; i < 5; i++) {
    a *= 1.5
  }
  return a
}

export const negativeMultiplicationAssignmentOperator = () => {
  let a = 1
  for (let i = 0; i < 5; i++) {
    a *= -2
  }
  return a
}

console.log(multiplicationAssignmentOperator())
console.log(decimalMultiplicationAssignmentOperator())
console.log(negativeMultiplicationAssignmentOperator())

File: python/basic_grammar/operation/assignment_operator/multiplication.py
def multiplication_assignment_operator():
    a = 1
    for i in range(0, 5):
        a *= 2
    return a


def decimal_multiplication_assignment_operator():
    a = 1
    for i in range(0, 5):
        a *= 1.5
    return a


def negative_multiplication_assignment_operator():
    a = 1
    for i in range(0, 5):
        a *= -2
    return a


print(multiplication_assignment_operator())
print(decimal_multiplication_assignment_operator())
print(negative_multiplication_assignment_operator())
